### DISCUSSION TITLE
Has not run for 24h+

### DISCUSSION THREAD

**Discussion by ThibaultVlacich** [renovatebot/renovate#8627] [Request Help]
I have an issue on one repository, and only one. Renovate has not been running in the last 24h.

The last run in the logs has an error "not-found" (probably happened because of [Yesterday's issues](https://twitter.com/githubstatus/status/1359175711416844291) with GitHub?), and since then there has not been any run.

The repository is: `ThibaultVlacich/lucklesslovelocks.com-api`

**Comment by rarkins** [comment]
Yes, this seems to have been caused by yesterday's problems. GitHub's API returned 404 for many repos, which led them to be disabled in Renovate. We'll fix it today
